activation/validation.go

```diff
@@ -203,57 +204,52 @@ func (*Validator) PostMetadata(cfg *PostConfig, metadata *types.PostMetadata) er
 func (v *Validator) VRFNonce(
 	nodeId types.NodeID,
 	commitmentAtxId types.ATXID,
-	vrfNonce *types.VRFPostIndex,
-	PostMetadata *types.PostMetadata,
+	vrfNonce uint64,
 	numUnits uint32,
 ) error {
-	if vrfNonce == nil {
-		return errors.New("VRFNonce is nil")
-	}
-
 	meta := &shared.VRFNonceMetadata{
 		NodeId:          nodeId.Bytes(),
 		CommitmentAtxId: commitmentAtxId.Bytes(),
 		NumUnits:        numUnits,
-		LabelsPerUnit:   PostMetadata.LabelsPerUnit,
+		LabelsPerUnit:   v.cfg.LabelsPerUnit,
```

Comment:
Similar topic as the one with nipost - I'd prefer if we not make this change. ATX v1 has `LabelsPerUnit` in it and we did validate that this value was present and the one we are expecting - we should not change ATX v1 validation just because v2 ATX are different.

Reply:
The value LabelsPerUnit in the ATX is still verified here: https://github.com/spacemeshos/go-spacemesh/blob/e7a1f1d90f9e55b9dc7f5fc682fcf44ee29d2c44/activation/validation.go#L91-L93 IMHO, this function doesn't need to be aware that ATX has a duplicated value for it.

Reply:
Reverted - now `labelsPerUnit` is passed as an argument (instead of `PostMetadata` struct).